QModManager/API/QModAPI.cs

```diff
@@ -1,4 +1,7 @@
-﻿using QModManager.Utility;
+﻿using QModManager.API.ModLoading;
+using QModManager.API.ModLoading.Internal;
+using QModManager.Utility;
+using System;
```

Comment:
Will check what happened here.  
If this happened because `global::` tags showed up in the usings, that may have been why.  
Otherwise, this is just cleanup to do.